README.md

```diff
@@ -45,14 +100,46 @@ npm install -g create-openapi-repo
 or use [`npx`](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b):
 
 ```bash
-npx create-openapi-repo <spec-root>
+npx create-openapi-repo <api-definition-root>
 ```
-- [Create GitHub repo](https://help.github.com/articles/create-a-repo/#create-a-new-repository-on-github) where your OpenAPI spec will live.
+
+### Starting without any OpenAPI definition
+
+- [Create a GitHub repo](https://help.github.com/articles/create-a-repo/#create-a-new-repository-on-github) where your OpenAPI definition will live.
 - [Clone your repo](https://help.github.com/articles/cloning-a-repository/) and execute the following command inside it:
 ```bash
-create-openapi-repo .
+npx create-openapi-repo
+```
+-  Commit and push your changes to the GitHub repo and follow instruction in the `README.md` of your newly created repo.
+
+### Starting with an OpenAPI definition
+
+```bash
+npx create-openapi-repo <api-definition-root>
```

Comment:
this is not accurate, the tool will ask you for existing definition in the process. Or maybe I should update it to work like ^

Reply:
Yes. I changed it in the current version.

It accepts two path params now to specify name of `openapi` and `docs` folders.

E.g you may run `npx create-openapi-repo openapi web` - this will create `openapi` and `web` (instead of `docs`) folders